### DISCUSSION TITLE
I am getting the below error

### DISCUSSION THREAD

**Discussion by patelvamshidhar** [learndevopsonline/learndevopsonline#1899] [Q&A]
![Screenshot_2024-05-22-12-47-21-78_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/learndevopsonline/learndevopsonline/assets/49507145/31bfd9f2-fc44-43eb-8b61-e5eb03066050)

**Comment by chaithanya1812** [comment]
Hi @patelvamshidhar 

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/63d15c6b-728b-47cd-85b8-fdc44250dfc8)

**Reply by patelvamshidhar** [reply]
Hi @chaithanya1812 

Thank you the above issue is resolved

**Comment by patelvamshidhar** [comment]
Hi @chaithanya1812
Today I got new error unable to find what is the mistake. Can you check this onces. 

![Screenshot_2024-05-23-12-56-16-28_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/learndevopsonline/learndevopsonline/assets/49507145/c36b69a8-fa0e-4672-8cc0-77e986bdac96)

![Screenshot_2024-05-23-12-56-20-13_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/learndevopsonline/learndevopsonline/assets/49507145/7bafeeca-eb66-4594-8d6c-f1971cdcffd1)

**Comment by chaithanya1812** [comment]
## Hi @patelvamshidhar, This is not accepted, posting screenshots from mobile phone.
### Can I get the ip address of Gocd Server.

**Reply by patelvamshidhar** [reply]
Sorry I am **Working from office** so I was unable to login **gocd in office laptop**.
My gocd server :-**http://44.201.190.170:8153/go/pipelines**

**Reply by chaithanya1812** [reply]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/1aab84bc-035e-4d23-971a-12baee25c8b2)

**Reply by patelvamshidhar** [reply]
I have removed the space after localhost getting the same error

**Reply by chaithanya1812** [reply]
### It's fine, good. Same Image one more error is there find it out easy?..

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/8b2c380f-1a4c-4d05-b50f-76c54556044c)

**Reply by patelvamshidhar** [reply]
Noo @chaithanya1812 i am not able to find the error 😊😊

**Reply by chaithanya1812** [reply]
## oky, It's ansible-pull not ansible-playbook

**Reply by patelvamshidhar** [reply]
@chaithanya1812 even I have  changed to ansible pull same error I am getting

**Reply by chaithanya1812** [reply]
No check once pipeline now it's fine, previously your changes are not reflected

**Reply by patelvamshidhar** [reply]
@chaithanya1812 **Thank you ** 😊😊
